bluesky/tests/test_run_engine.py

```diff
@@ -1742,8 +1743,8 @@ def plan():
             yield Msg('read', det)
             yield Msg('save')
         finally:
-            yield Msg('declare_stream', None, det2,  name='primary')
-            yield Msg('create', name='primary')
+            yield Msg('declare_stream', None, det2,  name='primary2')
+            yield Msg('create', name="primary2")
```

Comment:
Why primary2?

Reply:
@tacaswell This should indeed be a different name, the code fails here

https://github.com/bluesky/bluesky/blob/febaa1096b20ce4c883f719f72a08f4c07aba1ac/bluesky/bundlers.py#L150-L156

event-model already has a descriptor with name `"primary"` and so it rightly throws an error if another is made. 

This doesn't fail in current bluesky since the doc is made without this check:

https://github.com/bluesky/bluesky/blob/90bdda70877e4f825d877f83449e09e0e9eed89b/bluesky/bundlers.py#L157-L166

My most recent commit changes `"primary"` to `"det1"`, and `"primary2"` to `"det2"` to avoid confusion.